src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala

```diff
@@ -68,18 +69,28 @@ class LibraryApiServiceSpec extends BaseServiceSpec with LibraryApiService with
     """.stripMargin
 
   override def beforeAll(): Unit = {
+    consentServer = startClientAndServer(consentServerPort)
+
+    val consentPath = consentUrl.replace(FireCloudConfig.Duos.baseConsentUrl, "")
+    val consent = Consent(consentId = "consent-id-12345", name = "12345", translatedUseRestriction = Some("Translation"))
+    val consentError = ConsentError(message = "Unapproved", code = BadRequest.intValue)
+    val consentNotFound = ConsentError(message = "Not Found", code = NotFound.intValue)
 
-    workspaceServer = startClientAndServer(workspaceServerPort)
-    workspaceServer
-      .when(request.withMethod("GET").withPath(isCuratorPath))
-      .respond(
-        org.mockserver.model.HttpResponse.response()
-          .withHeaders(MockUtils.header).withStatusCode(OK.intValue)
-      )
```

Comment:
This workspace server was not in use. Additionally, it was mocking an orchestration path, not a rawls path.